.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+---
+
+## Checklist:
+
+- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
+  minor change.
+  - [ ] No
+    - [ ] The change in this pull request is minor.
+  - [ ] Yes
+    - Close #{issue_number}
+
+For the following questions, only check the boxes that are applicable.
+
+- [ ] Add descriptive pull request text.
+- [ ] Add/update relevant tests.
+- [ ] Add/update relevant documents.
+
+- Do you have triage permission of this repository?
+  - [ ] No
+    - You are good.
+  - [ ] Yes
+    - Does this pull request close an issue?
+      - [ ] No, this pull request also acts as an issue by itself.
+        - [ ] Assign yourself.
+        - [ ] Label this pull request.
+        - [ ] Put a milestone.
+        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
+        - [ ] Put this pull request under the appropriate epic on ZenHub.
+        - [ ] Put an estimate on ZenHub.
+        - [ ] Add dependency relationship with other issues/pull requests on ZenHub.
+      - [ ] Yes, this pull request addresses an issue and does not act as an issue.
+        - [ ] Connect this pull request to the underlying issue on ZenHub.
```

Comment:
Do you mean we should link the PR inside the issue or add the issue under `Linked Issues`(under the Milestone section of the PR)

Reply:
Yes, then they are tracked together

Reply:
Do you want to explicitly mention the `Linked Issues` section then, I think it would be clearer?

Reply:
oh ok I see it, it's on the bottom of the PR. The Linked Issues section also has the connected issue, maybe it does that automatically then after you use the Connect Issues section